python/examples/quickstarters/ai_productivity/main.py

```diff
@@ -11,15 +12,20 @@
 dotenv.load_dotenv()
 
 # Initialize tools.
-openai_client = OpenAI()
+openai_client = OpenAI(
+    base_url="https://oai.helicone.ai/v1",
+    default_headers={
+        "Helicone-Auth": f"Bearer {os.environ['HELICONE_API_KEY']}",
+        "Helicone-Cache-Enabled": "true",
+    },
```

Comment:
Consider adding a fallback mechanism in case Helicone service is unavailable:
```python
try:
    openai_client = OpenAI(
        base_url="https://oai.helicone.ai/v1",
        default_headers={
            "Helicone-Auth": f"Bearer {os.environ['HELICONE_API_KEY']}",
            "Helicone-Cache-Enabled": "true",
        },
    )
except Exception as e:
    logger.warning(f"Failed to initialize Helicone, falling back to direct OpenAI: {e}")
    openai_client = OpenAI()
```